templates/image-builder.yml

```
@@ -143,7 +143,7 @@ objects:
     - prefix: ${FLOORIST_QUERY_PREFIX}/builds
       query: >-
         select
-          job_id,created_at,org_id,account_number,email,request->>'distribution' as distribution,
+          job_id,created_at,org_id,account_number,email,request,image_name->>'distribution' as distribution,
```

```suggestion
          job_id,created_at,org_id,account_number,email,image_name,request->>'distribution' as distribution,
```
'distribution' needs to be queried on the request, not on image_name.

---

True, thank you! Fixed. :)